packages/ui-components/package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@cockroachlabs/icons": "^0.3.0",
     "@popperjs/core": "^2.4.3",
+    "final-form": "^4.20.1",
```

Comment:
Instead of bundling `final-form` with the lib, I think we should declare it as a `peerDependency`.

Reply:
done